spec/index.bs

```diff
@@ -4245,8 +4245,8 @@ enum GPUBlendFactor {
     "dst-alpha",
     "one-minus-dst-alpha",
     "src-alpha-saturated",
-    "blendcolor-component",
-    "one-minus-blendcolor-component"
+    "constant-component",
```

Comment:
"constant-component" reads weird. FWIW I still think "component" makes things more confusing. Maybe just "src / src-alpha" "one-minus-src/one-minus-src-alpha"?

Reply:
That sounds like a good suggestion!

Reply:
Will follow up